packages/tools/docs-tools/src/data/authors.json

```diff
@@ -0,0 +1,176 @@
+[
```

Comment:
hm I get why this file is in here.
but if we do that we want the assets in the tools as well.

But more optimal this needs to be in the docs repo.
This is data.
no idea how yet :)

Reply:
or we should copy it from docs to docs-tools on build time.
maybe that is best

Reply:
then I think it's better to do dynamic import instead of copying it.